### PR TITLE
Tdr 2466 ons body and series

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,15 @@ To undo what was done by the migrations, run `sbt flywayClean`
...
+* TDR Deploy Data Migration Lambda: this builds the migration code and deploys it
   to the Lambda which will run the migrations
-* TDR Database Migrations Run: this runs the latest version of the migrations
+* Run DB Migrations: this runs the latest version of the migrations
   lambda
 
-Migrations that have been merged to master will be run automatically on the Integration environment and can be run
-manually on other environments by the `TDR Database Migrations Deploy` Jenkins job. The job will:
+Migrations that have been merged to master will be deployed automatically on the Integration environment, and can be deployed
+manually on other environments with the `TDR Deploy Data Migration Lambda` GitHub action. The action will:
```
Comment (vimleshtna) [inline README.md:37]: Looks good, can you please update the `Publish generated code` section as well, thanks.

**Review by vimleshtna — COMMENTED** [review]
Looks good, just a minor comment, thanks.

**Review by vimleshtna — APPROVED** [review]
Looks good, thanks.

**Comment by TomJKing** [comment]
Still awaiting confirmation that this transfer is going ahead.

**Pull request closed without merging**